profiles/api/webhook.py

```diff
@@ -20,7 +21,11 @@ def _update(orcid: str) -> Response:
             raise NotFound(str(exception)) from exception
 
         access_token = orcid_tokens.get(profile.orcid)
-        orcid_record = orcid_client.get_record(orcid, access_token.access_token)
+
+        try:
+            orcid_record = orcid_client.get_record(orcid, access_token.access_token)
+        except RequestException as exception:
+            raise Forbidden(str(exception)) from exception
```

Comment:
Here we should delete the stored access token and then make a second request to update the profile from their public data.